console.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/python3
+"""Creates Console module for Airbnb Clone"""
+
+
+import cmd
+from models.base_model import *
+from models import storage
+
+class HBNBCommand(cmd.Cmd):
+    prompt = '(hbnb) '
+    
+    supported_classes = ["BaseModel", "User", "State",
+                         "City", "Amenity", "Place", "Review"]
+
+    blackList = ["id", "created_at", "updated_at"]
+    
+    def do_quit(self, line):
+        """Quit command to exit the program."""
+        return True
+
+    def do_EOF(self, line):
+        """Exit on EOF"""
+        return True
+
+    def emptyline(self):
+        """Do nothing on an empty line."""
+        pass
+    
+    def do_create(self, line):
+        """Creates a neew instance of BaseModel, saves and prints id"""
+        if not line:
+            print("** class name missing **")
+        else:
+            try:
+                cls = eval(line)()
+                cls.save()
+                print(cls.id)
+            except (NameError, AttributeError):
+                print('** class doesnt exist **')
```

Comment:
We'll want to convert this to check for the main class same as other actions, to avoid eval calling foreign code